scripts/login.js

```diff
@@ -0,0 +1,39 @@
+import { LOGIN_API_URL, POSTS_API_URL } from "./constants.js";
+import { fetchData } from "./fetchData.js";
+import { displayPosts } from "./utils/displayPosts.js";
+import { addToLocalStorage } from "./utils/localStorageUtil.js";
+
+const form = document.querySelector('#login-form');
+const email = document.querySelector('#login-email');
+const password = document.querySelector('#login-password');
+
+
+async function loginUser(user) {
+    try {
+        const postBody = JSON.stringify(user);
+        const userLoginData = await fetchData(LOGIN_API_URL, {
+            method: 'POST',
+            body: postBody,
+        }, false);
+        const token = userLoginData.accessToken;
+        addToLocalStorage('accessToken', token);
+        window.location.href = '../feed';
+    } catch (error) {
+        console.error('Error logging in');
```

Comment:
You need to display a message to the user in the html here, not just log the error.

Very basic example:

`document.querySelector("#message").innerHTML = '<div class="error">Wrong email or password</div>';`

Obviously you would need an element with id="message" in the html.

Reply:
Done